subprojects/plugins/src/main/java/org/gradle/jvm/application/tasks/CreateStartScripts.java

```diff
@@ -275,11 +274,11 @@ public void generate() {
         generator.generateWindowsScript(getWindowsScript());
     }
 
-    @Internal
+    @Input
     private Iterable<String> getRelativeClasspath() {
```

Comment:
Would you mind returning a `List` instead of an iterable? This way we would not run into desirialization problems for this task class since we would only serialize the `String`s, but not capture the implementation of the `Iterable/Collection`.